### ISSUE TITLE
Acknowledge Alerts

### ISSUE THREAD

**Issue by yooperbsn** [justdave/nwsweatheralertswidget#28]
Would it be possible to add an option to acknowledge or clear alerts once viewed?

Thanks,
Andrew

**Comment by justdave** [comment]
You mean like to make it disappear of the widget once you acknowledge that you've seen it?

**Comment by yooperbsn** [comment]
Right. Keep it in the list in the app, but be able to clear it from the widget. Would allow maximization of limited space in the widget. As you know, MI weather changes fast and so can our watches/warnings. Being able to clear them (until audible alerts are integrated) may be a good solution.

Andrew DeGraaf

________________________________

**Comment by yooperbsn** [comment]
Case in point, right now there is a flash flood warning in alcona county. Not pertinent to me in the thumb. I see the notification, I am aware of it, but can clear it to make room for additional messages.

Andrew DeGraaf

________________________________
